src/helpers/types.ts

```diff
@@ -5,6 +5,10 @@ export interface Query extends core.Query { }
 
 export interface Params extends core.ParamsDictionary { }
 
-export interface Request<ReqBody = any, ReqQuery = Query, URLParams extends Params = core.ParamsDictionary>
+export interface Request<
+ReqBody = any,
+ReqQuery = Query,
+URLParams extends Params = core.ParamsDictionary,
```

Comment:
Indentation? Maybe 2 spaces

Reply:
@ayocodingit

Reply:
yes, I'm running eslint fix, it's been like that from there

Reply:
cc @AldiNFitrah